### PR TITLE
November 18 Cloud release notes / doc changes

### PR DESCRIPTION
**Pull request by jwitz**
Tried to write these release notes based off of https://github.com/astronomer/astrohub-hq-manifests/pull/890, but it doesn't look like that PR contains all of the changes visible in staging. Is there another repo where we merged our Secrets functionality? 

Adding @chomipi88 for review in case there's any Data Plane changes in other repos that we should include here.

### CODE REVIEW

File: docs/environment-variables.md [hunk 1]
```diff
@@ -50,29 +48,25 @@ If you want to make changes to existing environment variables, you can edit the
...
+If you set environment variables in your Dockerfile, they are stored in the Docker image that is generated when you run `astro dev start` or `astro deploy`. If you have an environment variable with sensitive information or credentials, we strongly recommend setting the variable as a secret via the Astronomer Cloud UI instead of storing it in your Dockerfile.
 
 ## Set Environment Variables via Dockerfile
 
 If you want to store environment variables using an external version control tool, we recommend setting them in your `Dockerfile`. This file is automatically created when you first initialize an Airflow project on Astronomer via `astro dev init`.
 
-> **Note:** Given that this file will be committed upstream, we strongly recommend using a secrets protocol to store environment variables containing sensitive information.
+> **Note:** Given that this file will be committed upstream, we strongly recommend either storing sensitive variables via the Astronomer UI or using a 3rd party secrets protocol.
```
[the remark this answers is not in the document]
Comment (jwitz) [reply docs/environment-variables.md:69]: Was thinking this is more "caution"-y

File: docs/environment-variables.md [hunk 2]
```diff
@@ -30,49 +30,59 @@ Environment variables can be set directly in the Astronomer UI. To do so:
...
-    </div>
+    ![Green checkmark icon next to an updated value in the Environment Variables configuration table](/img/docs/variable-checkmark.png)
 
 5. Click **Update Variables** to apply your changes. This action restarts your Airflow Scheduler, Webserver, and Workers.
 
+:::info
+
+Once an environment variable key has been set, it cannot be changed. Only an environment variable's value can be modified.
+
+:::
```
Comment (paolaperaza) [inline docs/environment-variables.md:63]: I feel like this doesn't have to be an "info" note? Can just be below the steps?

File: docs/environment-variables.md [hunk 3]
```diff
@@ -30,49 +30,59 @@ Environment variables can be set directly in the Astronomer UI. To do so:
...
+:::caution
+
+Environment Variables that are set as secret can be modified, but the variable's secret value will never be shown to the user once it's been saved. To modify a secret environment variable, you'll be prompted to enter a new value.
+
+:::
+
 ### How environment variables are stored on Astronomer
 
-All values for environment variables that are set in the Astronomer UI are stored in a database managed by Astronomer as part of the "Control Plane". If you set environment variables in your Dockerfile, they are stored in the Docker image that is generated when you run `astro dev start` or `astro deploy`.
+All environment variables set from the Astronomer UI are stored in a database managed by Astronomer as part of the Control Plane. Values for secret environment variables are encrypted and stored in a managed volume such that they cannot be viewed in plain text by anyone, including Astronomer staff.
```
Comment (paolaperaza) [inline docs/environment-variables.md:73]:
```suggestion
All environment variables set from the Astronomer UI are stored in a database managed by Astronomer as part of the Control Plane. Values for secret environment variables are encrypted and stored in a managed secrets backend such that they cannot be viewed in plain text by anyone, including Astronomer staff.
```

File: docs/environment-variables.md [hunk 4]
```diff
@@ -30,49 +30,59 @@ Environment variables can be set directly in the Astronomer UI. To do so:
...
 
-If you have an environment variable with sensitive information or credentials, we recommend storing these values in an external secrets backend, such as [Vault](https://www.vaultproject.io/), instead of via the Astronomer UI or your Dockerfile.
+If you set environment variables in your Dockerfile, they are stored in the Docker image that is generated when you run `astro dev start` or `astro deploy`. If you have an environment variable with sensitive information or credentials, we strongly recommend setting the variable as a secret via the Astronomer Cloud UI instead of storing it in your Dockerfile.
```
Comment (paolaperaza) [inline docs/environment-variables.md:75]: This might actually be redundant to info in ## Set Environment Variables via Dockerfile? Up to u

**Review by paolaperaza — APPROVED** [review]

**Comment by chomipi88** [comment]
@jwitz No data plane changes with this release. There will be a number of them after Thanksgiving.

File: docs/environment-variables.md (hunk 3 above)
Comment (chomipi88) [reply docs/environment-variables.md:73]: I support the above suggestion. Volume seems to suggest its a virtual disk.

File: docs/release-notes.md [hunk 5]
```diff
@@ -6,14 +6,25 @@ id: release-notes
...
 
+## November 19, 2021
+
+### Secret Environment Variables
+
+You can now set secret environment variables via the Astronomer Cloud UI. The values of secret environment variables are hidden from all users in your Workspace, making them ideal for storing sensitive information related to your Astronomer projects. For more information, read [Set Environment Variables via the Astronomer UI](environment-variables#set-environment-variables-via-the-astronomer-ui).
+
+### Minor Improvements
+
+- You can now create new Clusters in `sa-east-1`.
```
Comment (paolaperaza) [inline docs/release-notes.md:25]:
```suggestion
- You can now create new Clusters in AWS `sa-east-1`.
```

File: docs/release-notes.md [hunk 6]
```diff
@@ -6,14 +6,25 @@ id: release-notes
...
+- Extra whitespace is now automatically removed from environment variables.
```
Comment (paolaperaza) [inline docs/release-notes.md:26]:
```suggestion
- Extra whitespace at the end of any environment variable set in the Astronomer UI is now automatically removed to ensure the variable is passed correctly.
```

**Review by paolaperaza — COMMENTED** [review]
2 minor tweaks here @jwitz

**Comment by chomipi88** [comment]
@jwitz I was mistaken... we do have one data plane change that snuck in as per https://github.com/astronomer/astrohub-hq-manifests/pull/894

We now support `sa-east-1` for the data plane

**Pull request merged**